File: PythonSolutions/level_one/age_guesses.py
def age_guess():
    day = 7
    print("Guess my birth day")
    guess = int(input())

    while True:
        if guess == day:
            print("Correct guess")
            break
        else:
            print("Incorrect Guess")
            print("Guess my birth day")
            guess = int(input())

if __name__ == '__main__':
    age_guess()

File: JavaSolutions/levelOne/AgeGuesses.java
package semicolonClassWork.levelOne;

import java.util.Scanner;

public class AgeGuesses {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int day = 7;
        System.out.println("Guess my birth day");
        int guess = input.nextInt();


        while (true){
            if (guess == day){
                System.out.println("Correct guess");
                break;
            } else {
                System.out.println("Incorrect Guess");
                System.out.println("Guess my birth day");
                guess = input.nextInt();
            }
        }
    }
}
